docs/concepts/multi-vector-search.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: "Multi-vector search"
+---
+
+In some cases, a single vector representation of your content is not enough to retrieve the most relevant results.
+For example, in a research paper database, you might want to:
+
+- Retrieve documents based on both the summary of the entire paper and summaries of individual paragraphs.
+- Rank results by combining scores from multiple embeddings.
+
+Let's define a collection with two [semantic indexes](/collections/create#semantic-index):
+
+<CodeGroup>
+
+```python Python
+from topk_sdk.schema import text, vector, semantic_index
+
+client.collections().create(
+    "papers",
```

Comment:
this should be call `paper_paragraphs` or something (maybe article instead of paper?) since its a collection of paragraphs not paers